Examples/CodePushDemoApp/package.json

```diff
@@ -7,16 +7,15 @@
 		"test": "jest"
 	},
 	"dependencies": {
-		"react": "16.0.0-alpha.12",
-		"react-native": "^0.47.1",
-		"react-native-code-push": "file:../../",
-		"react-native-windows": "^0.43.0-rc.0"
+		"react": "16.0.0-beta.5",
+		"react-native": "^0.49.3",
+		"react-native-code-push": "file:../../"
```

Comment:
`react-native-windows` is missing here

Reply:
@max-mironov I think we should land this PR first #1051

Reply:
I'd like to add this later, in another PR, so we keep previous version of react-native-windows in old `CodePushDemoApp-pre0.49` and we should test and upgrade to latest `react-native-windows` separately for newly created demo app.

Reply:
@max-mironov okay reasonable